readme.adoc

```diff
@@ -391,9 +391,14 @@ There is support for using `native-image` from GraalVM project to produce a bina
 
 Since not all java libraries can automatically be built with `native-image` - especially if using reflection feature are considered highly experimental.
 
-Just run `jbang --native helloworld.java` and `jbang` will use `native-image` from either `JAVA_HOME/bin` or `GRAALVM_HOME/bin` to
+Just run `jbang --native helloworld.java` and `jbang` will use `native-image` from either `$JAVA_HOME/bin` or `$GRAALVM_HOME/bin` or `$PATH` to
 produce a native image binary.
 
+The built native image binary will be in `~/.jbang/cache/jars/YOUR.java.HASH.jar.bin` (see [#508] for an `-o/--output`).
```

Comment:
```suggestion
The built native image binary will be placed in jbang's cache and used on next run.
```

I don't want to make the contents of ~/.jbang/cache "api" ...at leats for now it is to be considered a black box for users.

Definitely ok adding a `-o/--out` to make this possible to get..